cmd/dashboard.go

```diff
@@ -48,6 +50,18 @@ var clearDashboards = &cobra.Command{
 	},
 }
 
+// askForConfirmation prompts user to confirm operation
+func askForConfirmation(msg string) {
+	fmt.Printf(msg)
+	r := bufio.NewReader(os.Stdin)
+	ans, _ := r.ReadString('\n')
+	response := strings.ToLower(ans)
+	if response[0] != 'y' && response[0] != 'Y' {
```

Comment:
If we're toLower above I don't think we need to check both cases?

Reply:
fair point.